pandas/tests/io/json/test_pandas.py

```diff
@@ -4,11 +4,11 @@
 from pandas.compat import (range, lrange, StringIO,
                            OrderedDict, is_platform_32bit)
 import os
-
 import numpy as np
 from pandas import (Series, DataFrame, DatetimeIndex, Timestamp,
                     read_json, compat)
 from datetime import timedelta
+moto = pytest.importorskip("moto")
```

Comment:
This will skip the entire module if moto isn't found. I think some of these tests can run without moto, so do this skip inside each test that needs moto.

Reply:
this can be removed, moto is now always installed

Reply:
added as a regular import